File: day1/first.py
with open("input.txt") as file:
    content = file.read()
    lines = content.splitlines()
    lines.append('')
    print(lines)
    current, max_cal = 0, 0
    for line in lines:
        if not line.isdigit():
            if current > max_cal:
                max_cal = current
            current = 0
        else:
            n_line = int(line)
            current += n_line
    print(max_cal)

File: day1/first.rb
file = File.open('input.txt')
content = file.read
lines = content.split("\n")
current = 0
max_cal = 0
lines.each do |line|
  if line.match(/^(\d)+$/)
    n_line = Integer(line)
    current += n_line
  else
    max_cal = current if current > max_cal
    current = 0
  end
end
puts max_cal
